services/src/main/java/org/apache/druid/cli/CliMiddleManager.java

```diff
@@ -163,7 +164,8 @@ public WorkerNodeService getWorkerNodeService(WorkerConfig workerConfig)
           }
         },
         new IndexingServiceFirehoseModule(),
-        new IndexingServiceTaskLogsModule()
+        new IndexingServiceTaskLogsModule(),
+        new LookupModule()
```

Comment:
#7222 introduces `LookupSerdeModule` which I think might be nicer behavior for middle manager and overlord than using `LookupModule` and configuring it not to load, but it will require this PR to wait until the other is merged.

Reply:
@scrawfor #7222 was merged, can you please rebase when you get a chance?